docs/sidecar_configmap.md

```diff
@@ -8,11 +8,123 @@ Chaos Mesh runs a [fuse-daemon](https://www.kernel.org/doc/Documentation/filesys
 
 In sidecar container, fuse-daemon needs to mount the data directory of application by [fusermount](http://manpages.ubuntu.com/manpages/bionic/en/man1/fusermount.1.html) before the application starts.
 
-Most applications use different data directories, so you need to define the different sidecar configs for different applications.
+## How it works?
 
-## Configuration file
+Currently, Chaos Mesh supports two types of ConfigMaps:
 
-The following content is the sidecar ConfigMap defined for tikv:
+1. Template config. The skeleton of each sidecar config is similar, in order to fulfill different requirements and make the configuration simplified,
+Chaos Mesh supports creating common templates to be used by different applications.
+
+2. Injection config. This configuration will be combined with template config and finally generate a config to inject to the selected pods. 
+Since most applications use different data directories, volume name or container name, you can define different parameters based on the common template created in the first step.
+
+## Template Configuration
```

Comment:
Can we split `Template Configuration` to a single file? Because most users don't need to care about the details about this config and just need to know which name they need to set. We want most users to ignore this file.

Reply:
What do you mean by `split to a single file`? It is a single file now. Would you mind elaborating your idea?

Reply:
We can introduce the `Template Configuration`  in a single file and we just introduce the `Injection configuration` at this file, because we most users just need to care about `injection configuration`.

Reply:
Sorry I still cannot understand your idea. Do you want to put `Template config` and `Injection config` into one config file? If so, then this is out of scope of this PR because this is another feature request which we don't support yet. 

If you want that feature, you can open another issue to track that implementation.

Reply:
Just the document.

Reply:
Done. PTAL